fsharp-mode.el

```diff
@@ -404,8 +404,15 @@ folders relative to DIR-OR-FILE."
                               (file-name-directory dir-or-file)))
 
 (defun fsharp-mode/find-fsproj (dir-or-file)
-  (fsharp-mode-search-upwards (rx (0+ nonl) ".fsproj" eol)
-                              (file-name-directory dir-or-file)))
+  (let ((dir-name (file-name-directory dir-or-file)))
+    (or (fsharp-mode/find-project-json dir-or-file)
```

Comment:
I think this call and the method definition below can be removed right? It looks like you have inlined the search upwards into this method.